static/locales/de-DE.yaml

```
@@ -155,7 +155,7 @@ Settings:
   Player Settings:
     Player Settings: Videoplayer Einstellungen
     Force Local Backend for Legacy Formats: Erzwinge lokales System für Legacyformat
-    Remember History: Speichere Verlauf
+    Remember History: Speicher Verlauf
```

```suggestion
    Remember History: Verlauf speichern
```
Might be even better and more like e.g. Browsers .